lib/fluent/plugin/in_kubernetes_objects.rb

```diff
@@ -72,6 +72,9 @@ class KubernetesObjectsInput < Fluent::Plugin::Input
 
       desc 'A selector to restrict the list of returned objects by fields.'
       config_param :field_selector, :string, default: nil
+
+      desc 'The interval at which the objects will be watched.'
+      config_param :interval, :time, default: 15 * 60
```

Comment:
Hi, I want to learn from your code, and your experience, towards handling watch resumption in kubeclient itself (https://github.com/abonas/kubeclient/issues/273).

I'm curious why you chose to wait 15 minutes here before attempting to re-watch.
My impression is recent k8s versions using etcd3 only keep old resourceVersion around for ~5min, so would expect this to fail with "410 Gone" errors.  Have you seen those?

[I'm aware this code has been superceded by #34 which now immediately loops to watch again.  Just curious about the story :-]